packages/server/src/internalHandlers/account.ts

```diff
@@ -65,6 +65,7 @@ export const request: RPCHandler<AccountRequest["result"]> = async (
 
   return {
     rawAccount: serializeAccount(account),
+    parentAccountId: account.parentAccountId,
```

Comment:
You should probably update `serializeAccount`

Reply:
Done